### PR TITLE
Updated Scantrad presence

### PR DESCRIPTION
**Pull request by Kozou4ever**
Hey! I edited the Scantrad presence since they changed their DOM
I took the opportunity to add some features: 
-Buttons added to read the chapter the user is reading
-Forum pages added

I also write the presence in french since it is a french website and most of the users are french. 
Let me know if there are some changes to make.

### CODE REVIEW

File: websites/S/Scantrad/presence.ts [hunk 1]
```diff
@@ -1,42 +1,77 @@
...
+            fullMangaName = mangaName.split('-').join(' ');
+
+        function titleCase (title: string) {
+            var arr = title.split(' ');
+            var result = arr.map(
+                function(value: any) {
+                    return value.replace(value.charAt(0), value.charAt(0).toUpperCase());
+                });
+            return result.join(' ')
+        };
```
Comment (ririxidev) [inline websites/S/Scantrad/presence.ts:44]:
```javascript
string.charAt(0).toUpperCase() + string.slice(1);
```
?

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
and resolve conflicts

**Comment by ririxidev** [comment]
conflicts again xD

**Comment by ririxidev** [comment]
image
bc semid was probably doing stuff again

**Comment by ririxidev** [comment]
Okay do this; Close this pull request - do re-fork so you will have all up-to-date and then do changes to your presence 👍

**Comment by Kozou4ever** [comment]
Ok I google what were those conflicts and how to resolve them, and I think it's good

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix merging conflicts

**Review by Bas950 — APPROVED** [review]

**Pull request merged**